src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -11,7 +11,7 @@
 function jcms_notifications_drush_command() {
   $items = array();
   $items['article-import'] = array(
```

Comment:
`import` vs `ingest` does not differentiate between what the two tasks are doing, please rename to `article-import` (already correct) and `article-import-all`